### ISSUE TITLE
Porting to Firefox

### ISSUE THREAD

**Issue by oscarjn** [akumidv/tradingview-assistant-chrome-extension#74]
Hi all,
I was wondering if this great extension could be ported to Firefox and how difficult would that be.

I am asking this because; first - Firefox runs circles around any Chrome-based browser. Secondly, because of 'Aw, Snap!' out of memory error that is a well known bug and it will happen in Chrome-based browser, given enough time. Physical RAM is not a problem as I am running this on a machine with 64 GB of RAM. Brave browser seems to be the most reliable but it will still crash.

Waterfox browser (Firefox fork) allows you to install and run chrome extension but for some reason it can't install this extension. I have tried some other solutions like Foxify, to no avail.

Now with the newly added features in v1.17 this extension is a swiss army knife and it wold be awesome to have it running in a faster, more reliable environment.

Thanks.

**Comment by akumidv** [comment]
I think it demand hundred hours for porting to Firefox and it would demand support both version.  This is a lot for an open source product.

But by memory error, I can try to optimize this in future, I need to figure out in what reason of problem there. Now all the test results are stored in memory, there may be an error with a very large number of tests for a large number of parameters. For example, more than 2000 or 5000 for a large strategy. If there is more information, I can possible improve this in one of the versions.

**Comment by oscarjn** [comment]
Yes you are correct, I am running it on a large number of cycles (<5000). Coincidentally, it crashed today after exactly 2000 cycles lol, but to be honest it seems to be random.

I may be wrong but I have the feeling there's nothing one can do to improve the extension, is just how Chrome works. Thanks for giving a thought, I was hoping anyone knows a quick solution like Chrome Store Foxified used to be back in the days.

Regards.

**Comment by akumidv** [comment]
I've found and fixed problem with events. I think it dramatically removed Chrome memory usage and possible increase speed of backtesting. So may be you do not need proting to Firefox. This changes present in current v.2.0 in main branch and today sent to review and publish in google webstore.

Can you check?
